arangod/Aql/EngineInfoContainerDBServerServerBased.cpp

```diff
@@ -379,8 +379,7 @@ Result EngineInfoContainerDBServerServerBased::buildEngines(
     if (!result.ok()) {
       return result;
     }
-    auto insert = serverToQueryId.try_emplace(serverDest, globalId);
-    TRI_ASSERT(insert.second);
+    serverToQueryId.emplace_back(serverDest, globalId);
```

Comment:
micro optimization: we can make `serverDest` non-const and move it into `serverToQueryId`.
```suggestion
    serverToQueryId.emplace_back(serverDest, globalId);
```